### ISSUE TITLE
Website shows as html source

### ISSUE THREAD

**Issue by rocketnik** [krzysztofzablocki/SourceryPro-Feedback#15]
If you got the menu Help -> Documentation you will be redirected to the website
https://cdn.jsdelivr.net/gh/krzysztofzablocki/Sourcery@master/docs/index.html

Unfortunately it displays as html source. See image:

<img width="564" alt="image" src="https://user-images.githubusercontent.com/8329809/155083099-29844476-47e0-47d2-b158-f804f097d3ff.png">

It would be cool, if it displayed as rendered html.

Also I would suggest to add to the Help menu the menu item `Stencil Documentation` with a link to
https://stencil.fuller.li/en/latest/builtins.html (or some similar page with a definition what's possible to write)

**Comment by krzysztofzablocki** [comment]
Yeah that's fair, the doc link changed between updates, it's now here http://merowing.info/Sourcery/
